src/app.jsx

```diff
@@ -98,12 +102,13 @@ export const Application = () => {
 
             channel.current.addEventListener("ready", () => {
                 setFiles(files);
+                setDirItemsCnt(itmCnt);
```

Comment:
This won't happen for watch events, so the count will become wrong with added files. This should get a test case. In general, the whole counting functionality doesn't have tests, and this bug shows that they are needed. :grin:

Reply:
Oh good catch - indeed - with file removal this breaks.